### PR TITLE
[Proposal] Make docs translatable

### LINKED ISSUE #302: [Proposal] Make docs translateable
In order to clean up the controllers and to handle summaries, descriptions etc. better it would be beneficial to store them into one file.
An easy way to achieve this is to make said descriptions etc. translatable. 

As a bonus, the doc and the descriptions can be suited to the needs of customers.

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request
Resolves #302

### CODE REVIEW

File: translations/studio_api_docs.en.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  Update needs to have the complete data present. <br>
+  You can create/update/delete list entries like metadata, custom settings and properties. <br> 
+  E.g. if you want to remove an entry from metadata simply do not include this entry in the update. <br> 
+  If you want to update only a single field, use the PATCH method.
+cloneElement_summary: Clone a specific asset
+cloneElement_description:  > 
+  Clones a specific asset with the given id. <br>
+  The parentId must be a folder
+cloneElement_success_response: Successfully copied asset
+cloneElement_created_response: Successfully copied parent asset and created jobRun for copying child assets
```
Comment (lukmzig) [inline translations/studio_api_docs.en.yaml]: should these be `clone_Asset ....`?

Comment (mattamon) [reply translations/studio_api_docs.en.yaml]: you are totally right :D

Comment (mattamon) [reply translations/studio_api_docs.en.yaml]: It just took the operationId, maybe we should change it too?

Comment (lukmzig) [reply translations/studio_api_docs.en.yaml]: yeah lets change it to cloneAsset

**Review by lukmzig — APPROVED** [review]
LGTM! :seal:

**Pull request merged**